### DISCUSSION TITLE
Locally Hosted docker install error:  There was a problem loading quick start templates.

### DISCUSSION THREAD

**Discussion by ksorgIN** [Budibase/budibase#9931] [Help]
After following the configuration for a locally hosted install, When I get to the design portion of my app I receive the error There is a problem loading quick start templates.  I have started over and I am still receiving this error.  Any assistance would be great.

![image](https://user-images.githubusercontent.com/127224118/223488493-f618c2f0-8c34-472e-8d57-7ce63e593246.png)

**Comment by ksorgIN** [comment]
I also see this on the console of the server.

```
`budibase-couchdb-service-1     | [notice] 2023-03-07T19:17:10.111298Z couchdb@127.0.0.1 <0.363.0> -------- chttpd_auth_cache changes listener died because the _users database does not exist. Create the database to silence this notice.
budibase-couchdb-service-1     | [error] 2023-03-07T19:17:10.112363Z couchdb@127.0.0.1 emulator -------- Error in process <0.23476.0> on node 'couchdb@127.0.0.1' with exit value:
```
`

**Comment by ConorWebb96** [comment]
Hey @ksorgIN,

Your url looks a bit strange to me, normal setup would end with hitting localhost:10000. If you are looking to use that specific address you may need some port forwarding/tweaking of the docker-compose file.

**Reply by ksorgIN** [reply]
That is just the IP of the server connecting from a remote system.
